### PR TITLE
Despawn time

### PR DESCRIPTION
**Pull request by Rambolo**
Added toggleable despawn timers to the ESP for Ores and Forageables (big thanks to Diyagi for his effort & teachings). Also added a '+1' to the selectedSlot in the slot selection dropdown for user continuity (doesn't break anything).

the .gitignore file can be ignored, not sure how to remove that from the pull request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -368,4 +368,5 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
```
Comment (Rambolo) [inline .gitignore:371]: Ignore

**Comment by Wimberton** [comment]
Once we push for this update, we'll look at manually merging! Thanks again!

**Comment by Rambolo** [comment]
Crashing on gatherables reaching 0. Attempting to fix. Will update with code suggestion when I figure it out.

**Comment by Rambolo** [comment]
> Crashing on gatherables reaching 0. Attempting to fix. Will update with code suggestion when I figure it out.

Fixed and added suggestion.

**Comment by Wimberton** [comment]
Closing with previous pushes! Thanks again!

**Pull request closed without merging**